src/main/java/com/autotune/analyzer/workerimpl/BulkJobManager.java

```diff
@@ -0,0 +1,229 @@
+                //example metadataInfo = dataSourceManager.importMetadataFromDataSource(datasource, uniqueKey, interval_start_time_epoc, interval_end_time_epoc, steps);
+            } else {
+                //Get metaData
+                //metadataInfo = dataSourceManager.importMetadataFromDataSource(datasource, uniqueKey, 0, 0, 0);
+            }
+            List<String> recommendationsRequiredExperiments = new CopyOnWriteArrayList<>();
+            if (null == metadataInfo) {
+                jobStatusMap.get(jobID).setStatus("COMPLETED");
+            } else {
+                Collection<DataSource> dataSourceCollection = metadataInfo.getDataSourceHashMap().values();
+                for (DataSource ds : dataSourceCollection) {
+                    HashMap<String, DataSourceCluster> clusterHashMap = ds.getDataSourceClusterHashMap();
+                    for (DataSourceCluster dsc : clusterHashMap.values()) {
+                        HashMap<String, DataSourceNamespace> namespaceHashMap = dsc.getDataSourceNamespaceHashMap();
+                        for (DataSourceNamespace namespace : namespaceHashMap.values()) {
+                            HashMap<String, DataSourceWorkload> dataSourceWorkloadHashMap = namespace.getDataSourceWorkloadHashMap();
+                            if (dataSourceWorkloadHashMap != null) {
+                                for (DataSourceWorkload dsw : dataSourceWorkloadHashMap.values()) {
+                                    HashMap<String, DataSourceContainer> dataSourceContainerHashMap = dsw.getDataSourceContainerHashMap();
+                                    if (dataSourceContainerHashMap != null) {
+                                        for (DataSourceContainer dc : dataSourceContainerHashMap.values()) {
+                                            CreateExperimentAPIObject createExperimentAPIObject = new CreateExperimentAPIObject();
+                                            createExperimentAPIObject.setMode("monitor");
+                                            createExperimentAPIObject.setTargetCluster("local");
+                                            createExperimentAPIObject.setApiVersion("v2.0");
+                                            String experiment_name = "prometheus-1" + "-" + dsc.getDataSourceClusterName() + "-" + namespace.getDataSourceNamespaceName()
+                                                    + "-" + dsw.getDataSourceWorkloadName() + "(" + dsw.getDataSourceWorkloadType() + ")" + "-" + dc.getDataSourceContainerName();
+                                            createExperimentAPIObject.setExperimentName(experiment_name);
+                                            createExperimentAPIObject.setDatasource("prometheus-1");
+                                            createExperimentAPIObject.setClusterName(dsc.getDataSourceClusterName());
+                                            createExperimentAPIObject.setPerformanceProfile("resource-optimization-openshift");
```

Comment:
This should be `resource-optimization-local-monitoring` - https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_local_monitoring.json

Reply:
done